README.md

```diff
@@ -1 +1,42 @@
-# quarkus-hazelcast-client
+# Hazelcast Client for Quarkus
+
+<a href="https://github.com/actions/toolkit"><img alt="GitHub Actions status" src="https://github.com/hazelcast/quarkus-hazelcast-client/workflows/build/badge.svg"></a>
```

Comment:
It does not display correctly for me.
![image](https://user-images.githubusercontent.com/2834997/77886886-efe33980-7269-11ea-81cb-a223a17f27a6.png)